tests/test_gsc_bookmarks.py

```diff
@@ -1,41 +1,46 @@
-from datetime import timedelta
 from datetime import datetime as dt
-from tap_tester import connections, menagerie, runner
+from datetime import timedelta
+
 from base import GoogleSearchConsoleBaseTest
+from tap_tester import connections, menagerie, runner
 from tap_tester.logger import LOGGER
 
+
 class GoogleSearchConsoleBookMarkTest(GoogleSearchConsoleBaseTest):
-    """Test tap sets a bookmark and respects it for the next sync of a stream"""
+    """Test tap sets a bookmark and respects it for the next sync of a
+    stream."""
 
     @staticmethod
     def name():
         return "tap_tester_google_search_console_bookmark_test"
 
     def get_second_sync_bookmark_date(self):
-        return dt.strftime(dt.utcnow()-timedelta(days=7), "%Y-%m-%dT00:00:00.000000Z")
+        return dt.strftime(dt.utcnow() - timedelta(days=7), "%Y-%m-%dT00:00:00.000000Z")
 
     def test_run(self):
-        """
-        Verify that for each stream you can do a sync which records bookmarks.
-        That the bookmark is the maximum value sent to the target for the replication key.
-        That a second sync respects the bookmark
-            All data of the second sync is >= the bookmark from the first sync
-            The number of records in the 2nd sync is less then the first (This assumes that
-                new data added to the stream is done at a rate slow enough that you haven't
-                doubled the amount of data from the start date to the first sync between
-                the first sync and second sync run in this test)
-        Verify that for full table stream, all data replicated in sync 1 is replicated again in sync 2.
-        PREREQUISITE
-        For EACH stream that is incrementally replicated there are multiple rows of data with
-            different values for the replication key
+        """Verify that for each stream you can do a sync which records
+        bookmarks.
+
+        That the bookmark is the maximum value sent to the target for
+        the replication key. That a second sync respects the bookmark
+        All data of the second sync is >= the bookmark from the first
+        sync     The number of records in the 2nd sync is less then the
+        first (This assumes that         new data added to the stream is
+        done at a rate slow enough that you haven't         doubled the
+        amount of data from the start date to the first sync between the
+        first sync and second sync run in this test) Verify that for
+        full table stream, all data replicated in sync 1 is replicated
+        again in sync 2. PREREQUISITE For EACH stream that is
+        incrementally replicated there are multiple rows of data with
+        different values for the replication key
```

Comment:
Remove spacing on line no. 23, and remove unnecessary spacing in between the words. Keep PREREQUISITE on the next line.

Reply:
Done